pylint_odoo/test_repo/broken_module2/tests/data/odoo_data_noupdate_0_2.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+    </data>
+    <record>
```

Comment:
This file not should be raise the `deprecated-data-xml-node` check.
I go to work in that fix in another PR

Reply:
It did you work for oca
is not?

Reply:
@moylop260 
Yes, this is the another pull request https://github.com/OCA/pylint-odoo/pull/156 
In this pull request the false positive on `deprecated-data-xml-node` was fixed 

Sorry for not report this change